### PR TITLE
[1477] url changes panel

### PR DESCRIPTION
**Pull request by jremen**
**Changes proposed in this Pull Request**
URL changes panel comparator

Close QualityUnit/web-issues#1477
Close QualityUnit/web-issues#1645
Close QualityUnit/web-issues#1650

### CODE REVIEW

**Comment by ihdk** [comment]
@jremen 
Not sure if it's something that should be or will be resolved, so just ask:

[#1477](https://github.com/QualityUnit/web-issues/issues/1477) - shouldn't be edited generator result immediately reflected on edited row in table after save in popup? now, manual table refresh is necessary to see the change.

[#1645](https://github.com/QualityUnit/web-issues/issues/1645) - this works with invalidated modules query, maybe we can later remove unnecessray rerender and flashing of modules and header bottom after activation/deactivation...
I have it done locally with few simple changes using memoization and `fetchedModules` saved into store. Data are not passed deeply into `<MainMenu>` and `<DynamicModule>` unnecessary via props. 
Query invalidation cause rerender of whole `<DynamicModule>` thus flashing of modules and header bottom with fade in animation...

Except these things it can be approved, I do not have any comments...

File: admin/src/api/importCsv.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export default async function importCsv( { slug, dataArray, result, stopImport }
 	const chunksLength = dataChunks().length;
 	let chunkIndex = 0;
 	let ended = false;
+	result( 1 );
```
Comment (vzeman) [inline admin/src/api/importCsv.js:16]: @jremen naco je tento return? debugovaci? alebo ma zmysel?

**Review by vzeman — CHANGES_REQUESTED** [review]

File: admin/src/api/importCsv.js (hunk 1 above)
Comment (jremen) [reply admin/src/api/importCsv.js:16]: @vzeman
Aby po zaciatku importu necakalo na prvy response z Fetch ale hned zobrazilo na 1% progress bar

**Review by vzeman — APPROVED** [review]

**Comment by ihdk** [comment]
@vzeman sorac omylom som na mobile klikol request ked som si to tu prezeral co sa deje...

**Pull request merged**